src/DetailsPage/DetailsPage.js

```diff
@@ -23,18 +24,19 @@ class DetailsPage extends Component {
     return (
       <section className='DetailsPage'>
```

Comment:
Glad this is working now. Spent all night on it and couldn't get anything to work. Probably had some tunnel vision tbh.